packages/cozy-procedures/src/components/documents/DocumentHolder.jsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react'
 import { translate, Chip, Icon } from 'cozy-ui/transpiled/react/'
+import { CozyFile } from 'cozy-doctypes'
 
-const DocumentHolder = () => (
-  <Chip variant="dashed" className="u-w-100">
-    <Icon icon="file-type-pdf" size={24} className="u-mr-1" />
-    <span className="u-flex-grow-1">
-      Impots gouv 2019
-      <span className="u-coolGrey">.pdf</span>
-    </span>
-    <Icon icon="cross" size={16} className="u-pr-1" />
-  </Chip>
-)
+const DocumentHolder = ({ document }) => {
+  //!TODO to be removed. No need to check because the component should be mounted only when we have info
```

Comment:
That's the case now, isn't it?

Reply:
Done on the next PR (since here we still display an empty component https://github.com/cozy/cozy-libs/pull/542/files#diff-f8d3ccbe6490779f446ce1d9e0ebc62aL31)